frontend/src/metabase/components/TokenField.jsx

```diff
@@ -590,10 +590,7 @@ export default class TokenField extends Component {
       filteredOptions.length === 0 ? null : (
         <ul
           className={cx(optionsClassName, "overflow-auto pl1 my1 scroll-hide")}
-          style={{
-            ...optionsStyle,
-            maxHeight: this.props.parameter ? 300 : "none",
-          }}
+          style={{ maxHeight: 300, ...optionsStyle }}
```

Comment:
Reverted to https://github.com/metabase/metabase/pull/16308/files#diff-52dbad60907bc454d0a20c6b831ee0bb9d5e585d7ac0daab1f7b618f052d82a8L596